vulnerablecode/settings.py

```diff
@@ -7,95 +7,102 @@
 
 # Environment
 
-env_file = str(ROOT_DIR.joinpath(".env"))
-if Path(env_file).exists():
-    environ.Env.read_env(env_file)
+ENV_FILE = "/etc/vulnerablecode/.env"
+if not Path(ENV_FILE).exists():
+    ENV_FILE = ROOT_DIR / ".env"
```

Comment:
Shouldn't the local config be preferred over the one in ``/etc`` ?

Imagine a user with a bad ``/etc`` config deleting and recloning vulnerablecode a thousand times and expecting ``make env`` to fix their issue but it won't.

Reply:
> Shouldn't the local config be preferred over the one in /etc ?

not when doing a deployment. So in general the global is what matters on a server. The local is only for development use.

> Imagine a user with a bad /etc config deleting and recloning vulnerablecode a thousand times and expecting make env to fix their issue but it won't.

That's a good point... I should add some doc there. I also want to keep this in tight sync with scancode.io and avoid variations between the two unless there is a reason.  We may want to evolve a skeleton for this to keep them in sync

Reply:
I pushed some doc at https://github.com/nexB/vulnerablecode/pull/677/commits/bf4864bb88e54d13ee88e8e4322ec82eecbeedd3